src/graphql/url.generated.tsx

```diff
@@ -0,0 +1,82 @@
+import * as Types from "./types";
```

Comment:
nit: (and not needed for this PR):

The magic of making generated code like this not clutter up PRs by being unexpanded by default is by adding a file like this: https://github.com/brave/ads-admin-dashboard/blob/master/.gitattributes